src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionView.tsx

```diff
@@ -10,38 +17,54 @@ import {
     semVerOperators,
 } from 'constants/operators';
 import { useStyles } from '../ConstraintAccordion.styles';
+import {
+    PlaygroundConstraintSchema,
+    PlaygroundRequestSchema,
+} from '../../../../hooks/api/actions/usePlayground/playground.model';
 
 interface IConstraintAccordionViewProps {
-    constraint: IConstraint;
+    constraint: IConstraint | PlaygroundConstraintSchema;
```

Comment:
Maybe we could stick to one type here somehow, so that the this component doesn't have to know about the playground. This seems like a workaround.